### PR TITLE
Change require to show absolute path

### PR DESCRIPTION
**Pull request by skube**
As a best practice one should use absolute paths for Webpack stuff

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ Modules that are required for your style guide. Useful for third-party styles or
 module.exports = {
   require: [
     'babel-polyfill',
-    'path/to/styles.css',
+    '/absolute/path/to/styles.css',
```
Comment (sapegin) [inline docs/Configuration.md]: Would be better to show how to easily have an absolute path without hardcoding it:

```
path.join(__dirname, 'styleguide/styles.css'),
```

Comment (skube) [inline docs/Configuration.md]: Ok. Good idea!

**Comment by sapegin** [comment]
Thanks, merging!

**Comment by skube** [comment]
Thank you! 🥇

**Pull request merged**